dbt/profiles.yml

```diff
@@ -2,18 +2,18 @@ pudl_dbt:
   outputs:
     # Define targets for nightly build outputs, and local full/fast ETL outputs
     nightly:
-      threads: 4
+      threads: 1
```

Comment:
Default to single threaded execution until/unless we can figure out how to work around segfaults.